### PR TITLE
multisite tests from master, verify on slave

### PR DESCRIPTION
**Pull request by viduship**
Signed-off-by: viduship <vimishra@redhat.com>

# Description
Multisite tests:
Tests run from primary, verify on secondary.

This PR incorporates sanity multi-site test scenarios that will be triggered on the primary and would be verified on the secondary.

cephci logs:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1625748602312

### CODE REVIEW

File: suites/nautilus/rgw/sanity_rgw_multisite_2.yaml [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+                global:
+                  osd_pool_default_pg_num: 64
+                  osd_default_pool_size: 2
+                  osd_pool_default_pgp_num: 64
+                  mon_max_pg_per_osd: 1024
+              cephfs_pools:
+                - name: "cephfs_data"
+                  pgs: "8"
+                - name: "cephfs_metadata"
+                  pgs: "8"
```
Comment (psathyan) [inline suites/nautilus/rgw/sanity_rgw_multisite_2.yaml:87]: Are these pools required for running the tests?

File: suites/nautilus/rgw/sanity_rgw_multisite_2.yaml [hunk 2]
```diff
@@ -0,0 +1,372 @@
+tests:
```
Comment (psathyan) [inline suites/nautilus/rgw/sanity_rgw_multisite_2.yaml:1]: IMO, adding documentation about the objective of the test module helps in the long run. It also provides an overview to other team members.

**Review by psathyan — COMMENTED** [review]
Would it make sense to rename this file to `tier_2_rgw_multsite_primary.yaml`? We do have another suite `tier_1_rgw_multisite.yaml`...

**Comment by rakeshgm** [comment]
@viduship any update on the above requested changes

**Comment by viduship** [comment]
> Would it make sense to rename this file to `tier_2_rgw_multsite_primary.yaml`? We do have another suite `tier_1_rgw_multisite.yaml`...

This does not come under tier2 scenarios. The best name I can think of now would be tier_1_rgw_multisite_primary_to_secondary.yaml

File: suites/nautilus/rgw/sanity_rgw_multisite_2.yaml (hunk 1 above)
Comment (viduship) [reply suites/nautilus/rgw/sanity_rgw_multisite_2.yaml]: removed them

File: suites/nautilus/rgw/sanity_rgw_multisite_2.yaml (hunk 2 above)
Comment (viduship) [reply suites/nautilus/rgw/sanity_rgw_multisite_2.yaml]: Done.

File: suites/nautilus/rgw/tier_1_rgw_multisite_primary_to_secondary.yaml [hunk 3]
```diff
@@ -0,0 +1,375 @@
+"""
+Below are the multi-site test scenarios run on the master and verified the sync on the slave
+
+where :
+ceph-rgw1 is master/primary site
+ceph-rgw2 is slave/secondary site
+
+"""
```
Comment (psathyan) [inline suites/nautilus/rgw/tier_1_rgw_multisite_primary_to_secondary.yaml:8]: The syntax for comments is different for a yaml file. We would need to start the each line with `#`

Comment (rakeshgm) [reply suites/nautilus/rgw/tier_1_rgw_multisite_primary_to_secondary.yaml:8]: @viduship see this [yaml_comments](https://www.tutorialspoint.com/yaml/yaml_comments.htm) . if you had used an editor, the error would have come up instantly :)

Comment (viduship) [reply suites/nautilus/rgw/tier_1_rgw_multisite_primary_to_secondary.yaml]: Done

**Review by psathyan — APPROVED** [review]

**Pull request merged**